examples/resnet_vitis_ai_ptq_cpu/resnet_static_config.json

```diff
@@ -0,0 +1,95 @@
+{
+    "verbose": true,
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "model_path": "models/resnet_trained_for_cifar10.pt",
+            "model_storage_kind": "file",
+            "io_config": {
+                "input_names": ["input"],
+                "input_shapes": [[1, 3, 32, 32]],
+                "output_names": ["output"],
+                "dynamic_axes": {
+                    "input": {"0": "batch_size"},
+                    "output": {"0": "batch_size"}
+                }
+            }
+        }
+    },
+    "systems": {
+        "local_system": {"type": "LocalSystem"},
+        "aml_system": {
+            "type": "AzureML",
+            "config": {
+                "aml_config_path": "olive-workspace-config.json",
```

Comment:
Please remove the `aml_config_path` then update it with following config:

https://github.com/microsoft/Olive/blob/0d9c1f7260698659902452c8500e80823641da3b/examples/resnet_ptq_cpu/resnet_static_config.json#L3